src/equations/acoustic_perturbation_2d.jl

```diff
@@ -340,6 +340,71 @@ end
 end
 
 
+# Calculate 1D flux for a single point in the normal direction
+# TODO: Better comment, here this is really a normal vector
```

Comment:
Will this TODO (and the subsequent TODOs referring to comment) be fixed before the merge?

Reply:
Sure, I can do that. Is it okay with you and @ranocha if I name the variable `normal_direction` if it is not a normal vector?

Reply:
Yes :+1:

Reply:
Done.